integration/ui/satellite/onboarding_wizard_browser_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2021 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellite
+
+import (
+	"testing"
+	"time"
+
+	"github.com/go-rod/rod"
+	"github.com/go-rod/rod/lib/input"
+	"github.com/stretchr/testify/require"
+
+	"storj.io/common/testcontext"
+	"storj.io/storj/integration/ui/uitest"
+	"storj.io/storj/private/testplanet"
+)
+
+func TestOnboardingWizardBrowser(t *testing.T) {
+	uitest.Run(t, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet, browser *rod.Browser) {
+		signupPageURL := planet.Satellites[0].ConsoleURL() + "/signup"
+		fullName := "John Doe"
+		emailAddress := "test@email.com"
+		password := "qazwsx123"
+		page := browser.Timeout(20 * time.Second).MustPage(signupPageURL)
```

Comment:
Like in previous pull request this `Timeout(20 * time.Second)` could be redundant

Reply:
Yea, I realized it might be redundant but I just wanted to follow similar formatting to the first test UI login test.

Reply:
I see, then need to remove there as well, need to talk to @andriikotko

Reply:
Alright just added the new commit, with the your changes for this PR!